cumulusci/tasks/push/tests/test_push_api.py

```diff
@@ -24,6 +24,45 @@
 ORG_KEY = "bar"
 
 
+@pytest.fixture
+def metadata_package_versions(metadata_package):
+    return [
+        MetadataPackageVersion(
+            push_api=PUSH_API,
+            package=metadata_package,
+            name=NAME,
+            sf_id=SF_ID,
+            state="Beta",
+            major="3",
+            minor="1",
+            patch="1",
+            build="1",
+        ),
+        MetadataPackageVersion(
+            push_api=PUSH_API,
+            package=metadata_package,
+            name=NAME,
+            sf_id=SF_ID,
+            state="Beta",
+            major="2",
+            minor="1",
+            patch="1",
+            build="1",
+        ),
+        MetadataPackageVersion(
+            push_api=PUSH_API,
+            package=metadata_package,
+            name=NAME,
+            sf_id=SF_ID,
+            state="Beta",
+            major="1",
+            minor="1",
+            patch="1",
+            build="1",
```

Comment:
Would you mind adding some additional versions that use the other version numbers (`minor`, `patch`, etc.)?